### ISSUE TITLE
voice echo in call when both the devices use speakers

### ISSUE THREAD

**Issue by azizcreativitykw** [ConnectyCube/connectycube-cordova-samples#8]
hi

we are facing this issue while we use the speaker in both of our apps
if we use an earpiece then we don't hear our own voice but as soon as we turn on the speaker on both the devices, we start hearing out own voice

plz give us the appropriate solution asap
thanks

**Comment by ccvlad** [comment]
No solution. You are just using devices close to each other, so one of them hears and retranslate sounds from other and vice versa. It is normal.

**Comment by azizcreativitykw** [comment]
@ccvlad 

no, we are not keeping both the devices together
I mean to say if I keep my mobile on speaker
the sound coming from the speaker enters into a mic then the other person can hear their own voice twice
I hope you understand it

**Comment by ccvlad** [comment]
Yes, understand, but never met the same issue (Browser/Cordova/ReactNative). User cannot hear sound from self mic.
We will try to reproduce and think how to solve it if the problem will be reproduced. In any case, we will give an answer.

**Comment by azizcreativitykw** [comment]
@ccvlad 

I think, may be it's related to Acoustic Echo Cancellation

**Comment by ccvlad** [comment]
https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackSettings/echoCancellation

"Because RTP doesn't include this information, tracks associated with a WebRTC RTCPeerConnection will never include this property."

Can be tried but not sure what will help.
